SignalServiceKit/src/Messages/OWSMessageReceiver.m

```diff
@@ -105,27 +106,16 @@ - (instancetype)initWithDBConnection:(YapDatabaseConnection *)dbConnection
     return self;
 }
 
-- (NSArray<OWSMessageDecryptJob *> *)nextJobsForBatchSize:(NSUInteger)maxBatchSize
+- (OWSMessageDecryptJob *_Nullable)nextJob
 {
-    NSMutableArray<OWSMessageDecryptJob *> *jobs = [NSMutableArray new];
+    __block OWSMessageDecryptJob *_Nullable job = nil;
```

Comment:
As discussed offline, we can't batch decryption of messages - it risks data loss.  After decryption we have to persist the output ASAP to avoid data loss.